cmd/auth/auth.go

```diff
@@ -117,4 +120,6 @@ func init() {
 	AuthCmd.Flags().StringVarP(&password, "password", "p", "", "Backend AI password")
 	// add flag for url
 	AuthCmd.Flags().StringVarP(&baseURL, "baseurl", "u", "", "URL AI provider, (e.g `http://localhost:8080/v1`)")
+	// add flag for azure open ai engine/deployment name
+	AuthCmd.Flags().StringVarP(&engine, "engine", "e", "", "Azure AI deployment name")
```

Comment:
This needs to be a `MarkFlagRequired` right? It's not optional for this command

Reply:
thanks !
I've marked both engine and baseurl if backend is `azureopenai`